www/src/components/page-with-searchbar.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, Fragment } from "react"
```

Comment:
Probably we should rename this component to something like "plugin-sidebar"

Reply:
I agree that "searchbar" by itself is confusing. However, considering mobile view of plugins index (gatsbyjs.org/plugins) is just this "plugin-sidebar", would you be okay with "plugin-searchbar" instead?

Reply:
That sounds perfect.